cmd/gotemplate/gotemplate_test.go

```diff
@@ -37,7 +46,7 @@ func TestGenerate(t *testing.T) {
 	}{
 		"missing-file": {
 			in:          `{{include "no-such-file.txt"}}`,
-			expectedErr: "open no-such-file.txt: no such file or directory",
+			expectedErr: switchFileNotFoundError("no-such-file.txt"),
```

Comment:
I wonder if we can use ``errors.Is`` instead for checking what the error is in the test logic. Then, we can pass in fs.ErrNotExist as an argument here instead.

[1] https://pkg.go.dev/os#IsNotExist